### ISSUE TITLE
Redis server check fail

### ISSUE THREAD

**Issue by argosback** [gitlabhq/gitlabhq#9108]
Hi, when I run `gitlab:check` after update to v7.9.3 I get an error on the redis-server check

**Gitlab Setup**

```
# sudo -u git -H bundle exec rake gitlab:env:info RAILS_ENV=production

System information
System:         Ubuntu 14.04
Current User:   git
Using RVM:      no
Ruby Version:   2.1.5p273
Gem Version:    2.2.2
Bundler Version:1.8.5
Rake Version:   10.4.2
Sidekiq Version:3.3.0

GitLab information
Version:        7.9.3
Revision:       7ff7e28
Directory:      /home/git/gitlab
DB Adapter:     postgresql
URL:            http://git.example.com
HTTP Clone URL: http://git.example.com/some-project.git
SSH Clone URL:  git@git.example.com:some-project.git
Using LDAP:     no
Using Omniauth: no

GitLab Shell
Version:        2.6.0
Repositories:   /var/git/repositories/
Hooks:          /home/git/gitlab-shell/hooks/
Git:            /usr/bin/git

# redis-cli --version
redis-cli 3.0.0
```

**Check Fail**

```
# sudo -u git -H bundle exec rake gitlab:check RAILS_ENV=production
Redis version >= 2.0.0? ... no
  Try fixing it:
  Update your redis server to a version >= 2.0.0
  For more information see:
  gitlab-public-wiki/wiki/Trouble-Shooting-Guide in section sidekiq
  Please fix the error above and rerun the checks.
```

To pass these check I tweaked the regex at `lib/tasks/gitlab/check.rake:285`

```ruby
    def check_redis_version
      print "Redis version >= 2.0.0? ... "

      if run_and_match(%W(redis-cli --version), /redis-cli (2|3).\d.\d/)
        puts "yes".green
      else
        puts "no".red
        try_fixing_it(
          "Update your redis server to a version >= 2.0.0"
        )
        for_more_information(
          "gitlab-public-wiki/wiki/Trouble-Shooting-Guide in section sidekiq"
        )
        fix_and_rerun
      end
    end
```

**Comment by asiniy** [comment]
Same trouble. I use redis 3.0.0 but get this alert

**Comment by Delagen** [comment]
+1

**Comment by tonicbupt** [comment]
+1

so I'm gonna just ignore this...

**Comment by patschi** [comment]
+1 Have this same issue. But everything is still working fine. (7.10)

**Comment by starlessboi** [comment]
+1 same Issue (7.10)
but it seems everything is working normally

**Comment by vaziuok** [comment]
Fix that immediately!

**Comment by stanhu** [comment]
It's been fixed on master: 2e34b7aab5ad91bd. @Razer6, please close this one out.
